src/sonic_ax_impl/mibs/ietf/rfc4363.py

```diff
@@ -33,10 +45,21 @@ def fdb_vlanmac(self, fdb):
                     vlan_id = vlan_id.decode()
                 self.bvid_vlan_map[fdb["bvid"]] = vlan_id
         else:
-            return None
+            return None, None
         if not isinstance(vlan_id, str):
-            return None
-        return (int(vlan_id),) + mac_decimals(fdb["mac"])
+            return None, None 
+        return (int(vlan_id),) + mac_decimals(fdb["mac"]), int(vlan_id)
+
+    def get_tp_fdb_status(self, ent, fdb):
+        #TODO:add code for self type
+        if 'SAI_FDB_ENTRY_ATTR_TYPE' not  in ent:
+            return  TpFdbStatusConst.invalid
```

Comment:
>   [](http://example.com/codeflow?start=18&length=2)

Could you reformat new code? I see two blanks in this line and above line

Reply:
we will take care